modules/images/regenerate-existing-images/admin-page.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Admin page for background jobs
+ *
+ * @since n.e.x.t
+ * @package performance-lab
+ */
+
+/**
+ * Registers a management page for background jobs.
+ *
+ * @since n.e.x.t
+ */
+function perflab_register_background_jobs_page() {
+	add_management_page(
+		__( 'Background Jobs', 'performance-lab' ),
+		__( 'Background Jobs', 'performance-lab' ),
+		'manage_categories',
+		'background-jobs',
+		'perflab_render_background_jobs_page'
+	);
+}
+add_action( 'admin_menu', 'perflab_register_background_jobs_page' );
+
+/**
+ * Renders the background jobs page.
+ *
+ * @since n.e.x.t
+ */
+function perflab_render_background_jobs_page() {
+	$tax = get_taxonomy( 'background_job' );
+
+	/*
+	TODO: Uncomment this block once background_job capabilities are integrated with standard roles.
+	if ( ! current_user_can( $tax->cap->manage_terms ) ) {
+		wp_die(
+			'<h1>' . __( 'You need a higher level of permission.' ) . '</h1>' .
+			'<p>' . __( 'Sorry, you are not allowed to manage background jobs.' ) . '</p>',
+			403
+		);
+	}
+	*/
+
+	$screen            = get_current_screen();
+	$screen->post_type = 'post';
```

Comment:
Is this needed? Preferably the taxonomy shouldn't be attached to any specific post type.